src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/AutoCrafterListener.java

```diff
@@ -61,6 +74,12 @@ public void onInteract(PlayerRightClickEvent e) {
                 // Prevent blocks from being placed, food from being eaten, etc...
                 e.cancel();
 
+                // Check if the recipe of the item is disabled.
+                if (e.getPlayer().getWorld().getGameRuleValue(GameRule.DO_LIMITED_CRAFTING) && hasUnlockedRecipe(e.getPlayer(), e.getItem())) {
```

Comment:
Shouldn't this be inverted and check whether the Player has ***not*** unlocked the recipe :?

```suggestion
                if (e.getPlayer().getWorld().getGameRuleValue(GameRule.DO_LIMITED_CRAFTING) && !hasUnlockedRecipe(e.getPlayer(), e.getItem())) {
```

Reply:
Oof, my bad. One small sign change the whole thing, the entire the purpose of this PR.